### PR TITLE
edit, user

### PR DESCRIPTION
**Pull request by vuducluan**

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <div class="form-group">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "off", class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :current_password %> <i><%= t "current_password_to_change" %></i><br />
+      <%= f.password_field :current_password, autocomplete: "off", class: "form-control" %>
+    </div>
+    <div class="actions">
+      <%= button_to t("delete"),registration_path(resource_name), method: :delete,
```
Comment (hellovietnam93) [inline app/views/devise/registrations/edit.html.erb]: add space after ```,```

File: config/locales/en.yml [hunk 2]
```diff
@@ -28,3 +28,6 @@ en:
   sign_in: "Sign in"
   reset_password: "Forgot your password"
   sign_up: "Sign up"
+  update: "Update"
+  delete: "Delete Account"
+  alert: "Are you sure?"
```
Comment (hellovietnam93) [inline config/locales/en.yml]: name is not related to action.

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<h1><%= t "edit" %> <%= resource_name.to_s.humanize %></h1>
+<div class="col-md-6 col-md-offset-3">
+  <%= form_for resource, as: resource_name, url: registration_path(resource_name), html: {method: :put} do |f| %>
```
Comment (lyxuansang91) [inline app/views/devise/registrations/edit.html.erb]: quá 80 ký tự, thử cho xuống xem sao :)

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**